### PR TITLE
SCC Things

### PR DESCRIPTION
**Pull request by Geevies**
* Gave the CCIA a new SCC agent to spawn as. They need an admin to enable the slot, however.
* Added an energy repeater, given to SCC bodyguards.
* Added SCC undershirts and shorts.
* Added SCC jackets, shorts, flags, banners, floor decals, sleeve patches, armbands.
* CCIA ghostspawners have been put into a CCIA tab to declutter the admin tab.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 1]
```diff
@@ -23,6 +23,7 @@
 	..()
 	var/jackets = list()
 	jackets["bomber jacket"] = /obj/item/clothing/suit/storage/toggle/bomber
+	jackets["Stellar Corporate Conglomerate jacket"] = /obj/item/clothing/suit/storage/toggle/brown_jacket/scc
 	jackets["corporate black jacket"] = /obj/item/clothing/suit/storage/toggle/leather_jacket/nanotrasen
```
Comment (carpevenenum) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:27]: Shouldn't these be renamed into NanoTrasen jackets then, or is that best left alone until we actually change the map employer into SCC?

File: code/modules/ghostroles/spawner/human/admin.dm [hunk 2]
```diff
@@ -92,6 +92,38 @@
 	mob_name_prefix = "Ofc. "
 	mob_name_pick_message = "Pick a callsign or last-name."
 
+/datum/ghostspawner/human/admin/cciaagent/scc
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/admin.dm:95]: Keep the outfits out of standard access until the complete switchover from CCIA to SCC happens.
For now CCIA is still expected to handle the standard cases.
During non-standard cases admins can set the outfit as needed.

That said, if it really needs to be added, these paths are a abomination, turn them into something more readable akain to ccia is currently setup.
```
/datum/ghostspawner/human/admin/cciaagent/scc/bodyguard
```
Possibly turn both into subtypes of a parent type that cant be spawned in. ie.
```
/datum/ghostspawner/human/admin/corpagent/ccia
/datum/ghostspawner/human/admin/corpagent/scc
/datum/ghostspawner/human/admin/corpbodyguard/ccia
/datum/ghostspawner/human/admin/corpbodyguard/scc
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]
Did you check if the CCIA Recorder works with the new agent (title) ?
I think there is either a restriction on the rank and/or the title of the agent in the recorder.

**Comment by Geevies** [comment]
> 
> 
> Did you check if the CCIA Recorder works with the new agent (title) ?
> I think there is either a restriction on the rank (of the player) and/or the title of the agent in the recorder.

image
i believe this'll work, yeah

File: code/modules/ghostroles/spawner/human/admin.dm (hunk 2 above)
Comment (Geevies) [reply code/modules/ghostroles/spawner/human/admin.dm]: yeah, the team's planned to be able to spawn as SCC agents as soon as the SCC stuff hits the station, i believe. i organized the typepaths into its own folder, should look a lot better now

Comment (Geevies) [reply code/modules/ghostroles/spawner/human/admin.dm]: changed to admin edit

File: code/game/objects/structures/banner.dm [hunk 3]
```diff
@@ -121,4 +121,13 @@
 	name = "new gibson banner"
 	desc = "A banner depicting the flag of New Gibson."
 	icon_state = "newgibson_down"
-	icon_up = "newgibson_up"
\ No newline at end of file
+	icon_up = "newgibson_up"
+
+
+//////////////////////////////i'm adding a bunch of these slashes to look cool////////////////////////////
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/banner.dm:127]: No thanks.

File: code/game/objects/structures/banner.dm [hunk 4]
```diff
@@ -121,4 +121,13 @@
 	name = "new gibson banner"
 	desc = "A banner depicting the flag of New Gibson."
 	icon_state = "newgibson_down"
-	icon_up = "newgibson_up"
\ No newline at end of file
+	icon_up = "newgibson_up"
+
+
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/banner.dm:126]: Extra whitespace.

File: code/modules/projectiles/projectile/beams.dm [hunk 5]
```diff
@@ -30,6 +30,14 @@
 	damage = 25
 	armor_penetration = 10
 
+/obj/item/projectile/beam/pistol/scc
+	damage = 30
+	armor_penetration = 15
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/beams.dm:35]: Why is a pistol beam equal to a laser rifle shot?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/accessories/armband.dm [hunk 6]
```diff
@@ -78,4 +78,10 @@
 	desc = "An armband tailored to look like the flag of the Republic of Biesel."
 	desc_fluff = "While initially adopted during the early days of the TCFL to account for a sudden increase in volunteers and a lack of uniforms, during the height of the Republic of Biesel's conflicts with the Sol Alliance,\
 	it has been worn as a symbol of independence and patriotism."
-	icon_state = "tauceti"
\ No newline at end of file
+	icon_state = "tauceti"
+
+/obj/item/clothing/accessory/armband/tauceti
```
Comment (Arrow768) [inline code/modules/clothing/under/accessories/armband.dm:83]: Duplicate path

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by Geevies** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**